doc/rtd/topics/testing.rst

```diff
@@ -168,9 +146,9 @@ Test Argument Ordering
 .. [#fixture-list] This list of fixtures (with markup) can be
    reproduced by running::
 
-     py.test-3 --fixtures -q | grep "^[^ -]" | grep -v '\(no\|capturelog\)' | sort | sed 's/.*/* ``\0``/'
+     python3 -m pytest  --fixtures -q | grep "^[^ -]" | grep -v '\(no\|capturelog\)' | sort | sed 's/.*/* ``\0``/'
```

Comment:
A few comments on this one-liner:

Presumably the "no" is to remove `no tests ran in 0.00s`, but this filters out `no_cover` on more recent pytest versions - future-proof with longer match string.

"capturelog" doesn't filter anything on pytest 6.2.5, I assume this is something old that isn't needed?

It looks like "[session scope]" is being manually removed after running the command. This can be updated with a small updated to the `sed` command.

Assuming others agree with the above comments, this is what I propose:
```
python3 -m pytest  --fixtures -q | grep "^[^ -]" | grep -v 'no tests ran in' | sort | sed -e 's/ \[session scope\]//g;s/.*/* ``\0``/g'
```

Reply:
Works for me, thanks!